src/components/table-of-contents/index.jsx

```diff
@@ -21,7 +21,7 @@ export default function Toc() {
 
 	return (
 		<div ref={ref}>
-			<nav tabIndex={0} onFocus={this.open}>
+			<nav onFocus={this.open}>
```

Comment:
There's no real reason a user would need to tab onto the nav container, there's nothing for them to do there.